molecule/default/prepare.yml

```diff
@@ -16,8 +16,10 @@
     - name: "Download and deploy JWS zips from {{ down_server }}"
       ansible.builtin.get_url:
         url: "{{ item }}"
-        dest: "{{ tomcat_install_dir }}"
+        dest: "{{ jws_install_dir }}"
         validate_certs: no
       loop:
         - "{{ down_server }}/webserver/5.6.0/jws-5.6.0-application-server.zip"
         - "{{ down_server }}/webserver/5.6.0/jws-5.6.0-application-server-RHEL8-x86_64.zip"
+      become: yes
+      become_user: root
```

Comment:
This all section can be nuked. Downstream should now use the JBoss Network API and upstream can directly download from Apache TC website.